plugin/src/main/groovy/org/openbakery/XcodeBuildArchiveTask.groovy

```diff
@@ -438,6 +441,13 @@ class XcodeBuildArchiveTask extends AbstractXcodeBuildTask {
 
 	}
 
+    def removeUnneededDylibsFromBundle(File bundle) {
+        File libswiftRemoteMirror = new File(bundle, "libswiftRemoteMirror.dylib")
```

Comment:
Is that the only possible `dylib` to happen?

If that used to be done during packaging task, should it be removed from here?

Reply:
It seems to be the only one